iphone/Classes/ContactsModule.m

```diff
@@ -231,7 +231,8 @@ -(void)requestContactsPermissions:(id)args
         TiThreadPerformOnMainThread(^(){
             CNContactStore *ourContactStore = [self contactStore];
             [ourContactStore requestAccessForEntityType:CNEntityTypeContacts completionHandler:^(BOOL granted, NSError *error) {
-                NSDictionary * propertiesDict = [TiUtils dictionaryWithCode:[error code] message:[TiUtils messageFromError:error]];
+                NSString* errorMessage = (granted) ? (nil) : (@"The user has denied access to the address book");
+                NSDictionary * propertiesDict = [TiUtils dictionaryWithCode:[error code] message:errorMessage];
```

Comment:
This error msg is part of api response and hence not supposed to be displayed to end user. Because, we can essentially change this to a framework error msg in future when OS returns it to be.